src/modules/file/file.repository.ts

```diff
@@ -269,13 +274,17 @@ export class SequelizeFileRepository implements FileRepository {
   async getFilesWhoseFolderIdDoesNotExist(
     userId: File['userId'],
   ): Promise<number> {
+    const folders = await this.fileModel.findAll({
```

Comment:
This could be a lot of ids - not scalable - and I think this code is not related to the task, I will avoid changing these things